cypress/integration/login.js

```diff
@@ -1,103 +1,110 @@
+import LoginPage from '../objects/loginPage';
+import Common from '../objects/common';
+
+const loginPage = new LoginPage();
+const common = new Common();
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible');
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible');
     })
 
     it('Should be able to login with role Team Lead', function () {
 
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Ieva Stonkaitė"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Team Lead"]').click();
-        cy.get('[type="submit"]').click();
+        loginPage.visit();
+        common.getUserDropDown().click({force:true});
+        common.getSpecificUserFromDropDown("Ieva Stonkaitė").click();
+        common.getRoleDropDown().click({force:true});
+        common.getSpecificRoleFromDropDown("Team Lead").click();
+        loginPage.getSubmitButton().click();
 
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Ieva Stonkaite');
-        cy.get('.main-nav').find('li').should('have.length', 2);
+        cy.wait(2000); // for reason js sometimes doesn't load my client

```

Comment:
what is this about? it seems it work for others, try switching places with  common.getPageTitle().contains('Timesheets');

then it would wait for page title to change and afterwards assert URL, should be more stable that way, as URL changes during page loading and this might throw off cypress assert